Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Base on offical Node.js Alpine image
+FROM node:current-alpine AS base
+
+# Set working directory
+WORKDIR /base
+
+# Copy package.json and package-lock.json before other files
+# Utilise Docker cache to save re-installing dependencies if unchanged
+COPY package*.json yarn.lock ./ 
+
+# Install dependencies
+RUN yarn install --frozen-lockfile
+
+# Copy all files
+COPY . .
+
+# Build app
+FROM base AS builder
```

Comment:
Since above we add the comment to explain each step, would be nice to have it here too 🥰

Reply:
Of course, I will do the same as https://github.com/Cobda/cobda-web/pull/17#discussion_r575689629